warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryPruningVisitor.java

```diff
@@ -573,7 +573,7 @@ private void replaceAndAssign(JexlNode toReplace, String queryString, JexlNode b
                         }
                     }
                     
-                    JexlNodes.children(parent, children.toArray(new JexlNode[children.size()]));
+                    JexlNodes.children(parent, children.toArray(new JexlNode[0]));
```

Comment:
this is a change in node allocation size

Reply:
See previous comment about zero sized toArray calls